tests/utils.py

```diff
@@ -881,19 +891,11 @@ def make_data(
     def make_field_data(monitor: td.FieldMonitor) -> td.FieldData:
         """make a random FieldData from a FieldMonitor."""
         field_cmps = {}
-        coords = {}
         grid = simulation.discretize_monitor(monitor)
-
         for field_name in monitor.fields:
-            spatial_coords_dict = grid[field_name].dict()
-
-            for axis, dim in enumerate("xyz"):
-                if monitor.size[axis] == 0:
```

Comment:
This logic seems to be missing from your changes. Just to be careful I would try to include this behavior.

Reply:
This should be accounted for via the `len(coords) > 1` check I think

Reply:
The `monitor.center` is set by the user, so it might not match the cell locations. When the size is 0 along one dimension the monitor will default to interpolating along that coordinate.

Reply:
I see, ok makes sense. I restored the old behavior.